### PR TITLE
fix: raydium protocol fee adjustment

### PR DESCRIPTION
**Pull request by g3ndo9kar9**
## In This PR
- I correct the logic for raydium to account for the protocol pnl per raydium pool, specifically,
    - We deduct protocol pnl from the reserves (base/quote)
    - Add all open orders to respective liquidity reserve (base / quote)
    - finally derive a price given these new reserves

### CODE REVIEW

File: providers/apis/defi/raydium/price_fetcher.go [hunk 1]
```diff
@@ -250,7 +289,13 @@ func (pf *APIPriceFetcher) Fetch(
 			}
 			continue
 		}
-
+		pf.logger.Info(
```
Comment (Eric-Warehime) [inline providers/apis/defi/raydium/price_fetcher.go]: Move to debug please

File: providers/apis/defi/raydium/price_fetcher.go [hunk 2]
```diff
@@ -263,7 +308,7 @@ func (pf *APIPriceFetcher) Fetch(
 			metadata.BaseTokenVault.TokenDecimals, metadata.QuoteTokenVault.TokenDecimals,
 		)
 
-		pf.logger.Debug(
+		pf.logger.Info(
```
Comment (Eric-Warehime) [inline providers/apis/defi/raydium/price_fetcher.go]: Probably still needs to be debug

File: providers/apis/defi/raydium/price_fetcher.go [hunk 3]
```diff
@@ -238,8 +270,15 @@ func (pf *APIPriceFetcher) Fetch(
 			}
 			continue
 		}
+		pf.logger.Info(
```
Comment (Eric-Warehime) [inline providers/apis/defi/raydium/price_fetcher.go]: Debug

**Review by Eric-Warehime — APPROVED** [review]
LGTM after:
* Move info logs to debug
* Fix gofumpt
* Add comment to `amm_info` letting people know how the types were generated
* Fix docker builds

**Review by aljo242 — APPROVED** [review]

**Review by davidterpay — APPROVED** [review]

**Pull request merged**